sciserver_catalogs/catalog_xarray.yaml

```diff
@@ -1040,3 +1040,20 @@ sources:
         Sample (test) files
       projection: Mercator
       original_output: snapshot
+
+# ETOPO 2022
+  ETOPO:
+    description: ETOPO - Integrated topography, bathymetry, and shoreline data from regional and global datasets
+    driver: zarr
+    model: NA
+    args:
+      urlpath: '/home/idies/workspace/poseidon/data12_01/ETOPO'
+    xarray_kwargs:
+      engine: zarr
+      parallel: true
```

Comment:
Are there multiple files? Otherwise, `parallel=True` is not really necessary right?